database/tables/users.pg

```diff
@@ -6,7 +6,6 @@ columns
     lti_user_id: text
     name: text
     stripe_customer_id: text
-    terms_accepted_at: timestamp with time zone
```

Comment:
Why did you remove this?

Reply:
It was from update database description... unsure why it was deleted. I am on my phone at work but will look into it when I get to my laptop at home

Reply:
Resolved in recent commits.